### ISSUE TITLE
Question: Does a single leader address complexity sufficiently?

### ISSUE THREAD

**Issue by steffenkrogmann** [1729/content#39]
According to @balajis, the network state has a single definitive leader/founder. And a hierarchy follows from there.

In a strict hierarchy, the complexity profile of an organization is basically limited by the complexity profile of its leader.

How can we reconcile the need for an open network (vs. hierarchy) to better address complexity on the one hand, with the need for a leader (or someone/something) to set a vision and direction on the other hand?

This reading partially lead to that question:
https://necsi.edu/complexity-rising-from-human-beings-to-human-civilization-a-complexity-profile

**Comment by evanbranigan** [comment]
I agree. I think a network state needs leadership, but not necessarily a single leader

**Comment by Andreypnfrv** [comment]
Right now I'm building seed of network states and we use multi-leadership mechanisms to build. 

0. Board of Trustee, who are responsible for goal-setting
1. Single leader (CEO), who is guarantor for execution
2. C-Level Board, who are leaders of execution in specific fields

CEO can't fire C-Level without approval of BoT, can only run debates.

Decentralisation would come after launch is done.
But at first - centralisation helps to launch fast.

**Comment by steffenkrogmann** [comment]
Yeah, i guess it is not just about decentralization, but also about diversity (that word is being used so wrong these days^^) of opinion, skills, goals....
